datamigration.py

```diff
@@ -7,6 +7,8 @@
 from fabric.api import local, prompt  # pylint: disable=ungrouped-imports
 from fabric.contrib.console import confirm
 
+from os import path
+
```

Comment:
I am not at all unhappy with `path.join`, I wondered if anyone else shared my habitual preference and had a good reason for it :)